packages/pkgz/pkgz.pacscript

```diff
@@ -0,0 +1,26 @@
+pkgname="pkgz"
+arch=("all")
+pkgver="0.1.2"
+source=("@${pkgname}~${pkgver}::https://github.com/roguehashrate/pkgz/archive/refs/tags/v${pkgver}.tar.gz")
+pkgdesc="Fast, extensible CLI package manager written in Crystal"
+maintainer=("Your Name <youremail@example.com>")
+sha256sums=("e1ea57ac0bfa56c67d49d31f79179d5da5a5ed47e5ebcc136a06585ba39277bb")
+makedepends=("crystal" "gzip" "tar")
+external_connection=true
+
+build() {
+  # Check if shards is installed
+  command -v shards > /dev/null 2>&1 || {
+    echo "[ERROR] 'shards' not found. Please install shards before continuing."
+    exit 1
+  }
+
```

Comment:
```suggestion
```

No, we don't do this. Make sure your dependencies are actually installed and assume they are installed.